File: Python/uberBlack.py
from car import Car

class UberPool(Car):
    typeCarAccepted = []
    seatsMaterial = []
    
    def __init__(self, l, d, tCA, sM):
        super().__init__(l, d)
        self.typeCarAccepted = tCA
        self.seatsMaterial = sM

File: Java/UberBlack.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends Car {
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String l, Account d, Map<String, Map<String,Integer>> tCA, ArrayList<String> sM){
        super(l, d);
        this.typeCarAccepted = tCA;
        this.seatsMaterial = sM;
    }
}
